modules/Core/assets/prefabs/animTest.prefab

```diff
@@ -0,0 +1,12 @@
+{
+    "parent": "engine:iconItem",
+    "DisplayName": {
+        "name": "Animation Test"
+    },
+    "Item": {
+        "icon": "engine:items#whiteRecipe"
+    },
+    "InteractionScreen": {
```

Comment:
If you add `"InteractionTarget": {},` above this line then the UI opens and works again (along with the other changes)